lib/active_model_serializers/logging.rb

```diff
@@ -0,0 +1,112 @@
+##
+# ActiveModelSerializers::Logging
+#
+#   https://github.com/rails/rails/blob/280654ef88/activejob/lib/active_job/logging.rb
+#
+module ActiveModelSerializers::Logging
```

Comment:
I think the rubygems use this way in more contexts, I think the AMS codebase use the nested module in most cases. Do you think that make sense keep this exception or change to the nested?